config-model/src/main/java/com/yahoo/vespa/model/HostResource.java

```diff
@@ -22,17 +22,17 @@
  * TODO: Merge with {@link Host}
  * Host resources are ordered by their host order.
  *
- * @author lulf
- * @since 5.12
+ * @author Ulf Lillengen
  */
 public class HostResource implements Comparable<HostResource> {
 
     public final static int BASE_PORT = 19100;
     final static int MAX_PORTS = 799;
     private final Host host;
 
-    // Map from "sentinel name" to service
+    /** Map from "sentinel name" to service */
```

Comment:
*javadoc comment. Because it documents a specific variable. I agree it doesn't matter much because it is private though.